migration-config.conf.sample

```diff
@@ -72,6 +72,14 @@ network_map = external: external, yes: no
 # used for router migrations, if not set, tool assumes same name on destination
 # external_network_name_map = source_ext_net : dest_ext_net
 # storage_map = iscsi1:iscsi2
+#
```

Comment:
Please leave an uncommented line between each "major" config option (in this case please jsut remove the extra `#`)